member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.kt

```diff
@@ -0,0 +1,588 @@
+                        formatName(bankIdAuthResponse.givenName),
+                        formatName(bankIdAuthResponse.surname))
+                }
+            }
+            if (status == MemberStatus.INITIATED) {
+                applyChain = applyChain!!.andThenApply { MemberStartedOnBoardingEvent(id, MemberStatus.ONBOARDING) }
+            }
+        }
+        val authenticatedEvent = MemberAuthenticatedEvent(id, bankIdAuthResponse.referenceToken)
+        if (applyChain != null) {
+            applyChain.andThenApply(Supplier<Any> { authenticatedEvent })
+        } else {
+            apply(authenticatedEvent)
+        }
+    }
+
+    private fun getBirthdateFromSwedishSsnAndApplyEvent(ssn: String?, @Nullable applyChain: ApplyMore?): ApplyMore? {
+        var applyChain = applyChain
+        val birthDate = getBirthdateFromSwedishSsn(ssn)
+        if (birthDate != null) {
+            if (applyChain != null) {
+                applyChain = applyChain.andThenApply(
+                    Supplier<Any> { BirthDateUpdatedEvent(id, birthDate) }
+                )
+                return applyChain
+            }
+            apply(BirthDateUpdatedEvent(id, birthDate))
+        }
+        return applyChain
+    }
+
+    private fun generateTrackingId() {
+        apply(TrackingIdCreatedEvent(id, uuidGenerator.generateRandom()))
+    }
+
+    private fun formatName(name: String): String {
+        val lowercase = name.toLowerCase()
+        return Character.toUpperCase(lowercase[0]).toString() + lowercase.substring(1)
+    }
+
+    @Throws(RuntimeException::class)
+    private fun getPersonInformationFromBisnode(applyChain: ApplyMore?, ssn: String): ApplyMore? {
+        var chain = applyChain
+        logger.info("Calling bisnode for person information for {}", ssn)
+        val personList = bisnodeClient.match(ssn).persons
+        val person = personList[0].person
+        if (personList.size != 1) {
+            throw RuntimeException("Could not find person at bisnode.")
+        }
+        chain = chain!!.andThenApply { NameUpdatedEvent(id, person.preferredOrFirstName, person.familyName) }
+        val pi = BisnodeInformation(ssn, person)
+        if (pi.address.isPresent) {
+            chain = chain.andThenApply(
+                Supplier<Any> { LivingAddressUpdatedEvent(id, pi.address.get()) })
+        }
+        chain.andThenApply(Supplier<Any> { PersonInformationFromBisnodeEvent(id, pi) })
+        return chain
+    }
+
+    @CommandHandler
+    fun inactivateMember(command: InactivateMemberCommand?) {
+        if (status == MemberStatus.INITIATED || status == MemberStatus.ONBOARDING) {
+            apply(MemberInactivatedEvent(id))
+        } else {
+            val str = String.format("Cannot INACTIAVTE member %s in status: %s", id, status.name)
+            throw RuntimeException(str)
+        }
+    }
+
+    @CommandHandler
+    fun startOnboardingWithSSNCommand(command: StartOnboardingWithSSNCommand) {
```

Comment:
If we want to freshen up the naming here to align a bit more with how we actually deem this (especially with new developers coming in). Oh an @CommandHandlers usually are only named "handle" if want to clean this up as well.
```suggestion
    fun handle(command: StartSwedishOnboardingWithSSNCommand) {
```